packages/cardano-services/src/StakePool/TypeormStakePoolProvider/util.ts

```diff
@@ -63,32 +61,14 @@ export const nullsInSort = 'NULLS LAST';
 
 export const stakePoolSearchTotalCount = 'count(*) over () as total_count';
 
-export const getSortOptions = (
-  sort?: StakePoolSortOptions,
-  defaultField: SortField = 'name',
-  defaultOrder: SortOrder = 'asc'
-) => {
-  if (!sort) {
-    sort = { field: defaultField, order: defaultOrder };
-  }
-  if (sort.field === 'name') {
-    return {
-      field: `lower(${sortSelectionMap[sort.field as SortField]})`,
-      order: sort.order.toUpperCase() as Uppercase<SortOrder>
-    };
-  }
+export const getSortOptions = (sort?: StakePoolSortOptions) => {
+  if (!sort) sort = { field: 'name', order: 'asc' };
```

Comment:
Do we have any recommendation / convention to keep curly brackets `{...}` even for cases like this ☝️ It's a good practice to always put brackets.

Reply:
We're using eslint for stuff like this. It's currently valid since there's no rule enforcing brackets. If we want to change it, then a rule should be added and changes applied in the entire codebase.

Reply:
In line 67 there is `toUpperCase()` for order. Maybe it's good to put here `{ field: 'name', order: 'ASC'}` if we expect this to be uppercase?

Reply:
09a2b9584 should partially solve this.

About the case: in input `order` is of type
```typescript
export type SortOrder = 'asc' | 'desc';
```
while the output `order` property will be used in **TypeORM** `orderBy` method which accepts type `"ASC" | "DESC"`; that's why the function needs to make uppercase and why the default value for input should be expressed in lowercase.

Reply:
Disable eslint rule suggests that you should use a const/variable instead of that object literal :)

Reply:
f5e262f89 should fix this